GitUI/GitUICommands.cs

```diff
@@ -1168,8 +1168,8 @@ public bool StartBrowseDialog(IWin32Window? owner, BrowseArguments? args = null)
 
         public void StartFileHistoryDialog(IWin32Window? owner, string fileName, GitRevision? revision = null, bool filterByRevision = false, bool showBlame = false)
         {
-            string arguments = AppSettings.UseBrowseForFileHistory.Value ? $"browse {PathFilterArg}={fileName.Quote()} -commit={revision?.ObjectId}"
-                : $"{(showBlame ? BlameHistoryCommand : FileHistoryCommand)} {fileName.Quote()} {revision?.ObjectId} {(filterByRevision ? FilterByRevisionArg : string.Empty)}";
+            string arguments = AppSettings.UseBrowseForFileHistory.Value ? $"browse {PathFilterArg}={fileName.ToPosixPath().Quote()} -commit={revision?.ObjectId}"
```

Comment:
This could also be done in FormBrowse.cs:278
```
InitMenusAndToolbars(args.RevFilter, args.PathFilter.ToPosixPath());
```

Reply:
@gerhardol Happy to change it if that is best place! 🙂

Reply:
Updated!